applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -98,6 +125,19 @@ public static ConfigurationAsyncClientBuilder builder() {
     /**
      * Creates or updates a configuration value in the service with the given key.
      *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <pre>
+     * client.setSetting("prodDBConnection", "db_connection").subscribe(response -&gt; {
+     *     ConfigurationSetting result = response.value();
+     *     System.out.printf("Key: %s, Value: %s", result.key(), result.value());
+     * });
+     *
+     * client.setSetting("prodDBConnection", "updated_db_connection").subscribe(response -&gt; {
+     *     ConfigurationSetting result = response.value();
+     *     System.out.printf("Key: %s, Value: %s", result.key(), result.value());
+     * });</pre>
```

Comment:
In scenarios where you are demonstrating more than just how to use the code, such as here, you should add additional text that states the value of the `prodDBConnection` setting before either call, in the middle, and after the second call.